package.json

```diff
@@ -60,6 +60,10 @@
     "prettier": "^2.5.1"
   },
   "dependencies": {
+    "@types/dompurify": "^2.3.3",
+    "@types/markdown-it": "^12.2.3",
+    "dompurify": "^2.3.10",
+    "markdown-it": "^13.0.1",
```

Comment:
i dont believe we wanna have the dependencies in this package json -- instead we should abstract this into a a markdown component, thoughts @zomars ?

Reply:
we also need to test those packages for health

Reply:
Agreed @PeerRich we should check for weight, health, maintainability and overall complexity.

Reply:
@PeerRich removed `dompurify`, and we're now using `markdown-it` only on the server so client bundle size isn't affected.